postgres/grove/driver.go

```diff
@@ -0,0 +1,134 @@
+package grove
```

Comment:
As a reader, can you tell me what was copy-pasted from PADS so I don' t have to re-review?

Reply:
Can you call this out in the PR description?

Use github admonitions please.

https://github.com/orgs/community/discussions/16925

Reply:
OK, updated PR description. Done.